docs/content/connector-sink.md

```diff
@@ -105,6 +105,7 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 | sink.max-retries                  | No           | 3                 | The number of times that the system retries to perform the Stream Load job. This parameter is available only when you set `sink.version` to `V1`. Valid values: 0 to 10.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | sink.connect.timeout-ms           | No           | 30000             | The timeout for establishing HTTP connection. Valid values: 100 to 60000. Unit: ms. Before 1.2.9, the default value is 1000.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | sink.socket.timeout-ms            | No           | -1                | Supported since 1.2.10. The time duration for which the HTTP client waits for data. Unit: ms. The default value `-1` means there is no timeout.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| sink.sanitize-error-log          | No           | false             | When true, sensitive row data and column values in Stream-Load error logs are redacted in both connector and SDK logs. Intended for production security; defaults to false for backward compatibility. |
```

Comment:
```suggestion
| sink.sanitize-error-log          | No           | false             |  Supported since 1.2.12. When true, sensitive row data and column values in Stream-Load error logs are redacted in both connector and SDK logs. Intended for production security; defaults to false for backward compatibility. |
```